stylesheets/_component.rules.scss

```diff
@@ -0,0 +1,271 @@
+    text-transform: capitalize;
+
+    @include respond-min($screen-tablet) {
+        padding: 1.3em 0;
+        text-align: center;
+    }
+}
+
+.rule-block__heading__icon {
+    font-size: 16px;
+    margin-left: 5px;
+}
+
+.rule-block__controls {
+    margin: 1em 0;
+
+    @include respond-min($screen-tablet) {
+        margin: 1em 1.7em 0 .4em;
+    }
+}
+
+.rule-block__control {
+    display: block;
+    margin: 0 10px 10px;
+
+    @include respond-min($screen-tablet) {
+        display: inline-block;
+        margin: 0 0 .5em .4em;
+    }
+
+    .control__label {
+        padding: 0;
+    }
+}
+
+.conditional-block {
+    position: relative;
+}
+
+.conditional-block--flush {
+    // margin: 24px 0 24px 58px;
+    margin: 24px 0 0 58px; // seeing whether layout still works without bottom margin
+
+    @include respond-min($screen-tablet) {
+        // margin: 24px 0 24px 68px;
+        margin: 24px 0 0 68px; // seeing whether layout still works without bottom margin
+    }
+}
+
+.conditional-block--indented {
+    margin: 0 0 24px 75px;
+    padding-top: 14px;
+
+    &:before {
+        background: #DEDEDE;
+        content: '';
+        height: 26px;
+        left: 0;
+        position: absolute;
+        top: 0;
+        width: 4px;
+        z-index: 1;
+    }
+
+    @include respond-min($screen-tablet) {
+        margin: 0 0 14px 80px;
+    }
+}
+
+.conditional-block__link {
+    border: 0;
+    font-size: 1.2em;
+
+    i {
+        color: #DEDEDE;
+        display: inline-block;
+        margin-right: 5px;
+        text-decoration: none;
+    }
+
+    span {
+        text-decoration: underline;
+    }
+}
+
+td.table__form fieldset.rules__step {
+    margin-top: 0;
+}
+
+.form fieldset.rules__step legend,
+.piano__item-content .form fieldset.rules__step,
+.piano__item-content form > fieldset .rules__step legend {
+    border: 0;
+    margin: 0;
+    padding: 0;
+}
+
+.piano__item-content .rules-container .form {
+    padding-top: 12px;
+}
+
+.piano__item-content .rules-container .modal__content .form {
+    padding-top: 0;
+}
+
+td.table__form .rules + .form__group:last-of-type {
+    margin-bottom: 20px;
+
+    @include respond-min($screen-tablet) {
+        margin-bottom: 10px;
+    }
+}
+
+.rule-block__control.has-error {
+
+    .control__label {
+        color: $state-danger-text;
+    }
+
+    .form__control,
+    .select2-container .select2-selection--multiple,
+    .select2-container--default .select2-selection--multiple, {
+        background-color: $state-danger-bg;
+        border-color: color(danger);
+    }
+}
+
+.rule__triggers__collection .remove-button:hover,
+.rule__conditions__container .remove-button:hover,
+.rule__actions__container .remove-button:hover {
+    vertical-align: middle;
+}
+
+// scss-lint:enable all
+
+// Change default form layout styling
+.rule-block__control {
+
+    .form__group {
+        display: block;
+    }
+
+    .control__label {
+        display: block;
+        width: auto;
+    }
+}
```

Comment:
I'm guessing this has been copied from another place, is it worth running this through the scss-lint for the sake of 270 lines? It doesn't look like there is too much in there that would need changing